doc/sphinx-guides/source/admin/big-data-administration.rst

```diff
@@ -0,0 +1,312 @@
+- If the store has been configured with a remote-store-name or remote-store-url, the dataset file table will include this information for remote files. These provide a visual indicator that the files are not managed directly by Dataverse and are stored/managed by a remote trusted store.
+
+Challenges:
+
+- Currently, remote files can only be added via the API. (This may be addressed in future versions).
+- Remote files can only be added after the dataset is created in the UI (and therefore has an id and PID for use with the API). However, the UI will still allow upload of files to the base store (at dataset creation and when editing), which could be confusing.
+- As Dataverse is relying on the remote service to maintain the integrity and availability of the files, it is likely that the Dataverse site admin will want to have a formal agreement with the remote service 
+  operator about their policies.
+- Site admins need to consider carefully how to configure file size limits, ingest size limits, etc. on the remote store and it's base store, and whether the remote store is public-only, and whether files there can be read by Dataverse to assure the
+  requirements of a specific use case(s) are addressed.
+- The current remote store implementation will not prevent you from providing a relative URL that results in a 404 when resolved. (I.e. if you make a typo). You should check to make sure the file exists at the location you specify - by trying to download in Dataverse, by checking to see that Dataverse was able to get the file size (which it does by doing a HEAD call to that location), or just manually trying the URL in your browser.
+- For large files, direct-download should always be used with a remote store. (Otherwise the Dataverse will be involved in the download.)
+- When multiple files are selected for download, Dataverse will try to include remote files in the zip file being created (up to the max zip size limit) which is inefficient, and will not be able to include remote files that are inaccessible (possibly confusing). 
+
+Globus Stores: Globus Transfer between Large File/Tape Archives
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Note: Globus Transfer is still experimental: feedback is welcome! See :ref:`support`.
+
+`Globus <https://www.globus.org>`_ provides file transfer service that is widely used for the world's largest datasets (in terms of both file size and number of files). It provides:
+
+- robust file transfer capable of handling delays (e.g. due to the time it takes to mount tapes) and restarting after network or endpoint failures
+- rapid parallel file transfers, potentially between clusters of computers on both ends
+- third-party transfer, which enables a user working with their desktop browser to initiate transfer of files from one remote endpoint (i.e. on a local high-performance computing cluster) to/from another (e.g. one associated with a Dataverse store)
```

Comment:
```suggestion
- Robust file transfer capable of handling delays (e.g. due to the time it takes to mount tapes) and restarting after network or endpoint failures
- Rapid parallel file transfers, potentially between clusters of computers on both ends
- Third-party transfer, which enables a user working with their desktop browser to initiate transfer of files from one remote endpoint (i.e. on a local high-performance computing cluster) to another (e.g. one associated with a Dataverse store)
```

For consistency, I'd suggest capitalizing each bullet, like the other bullets in this doc.